docs/searchbar.md

```diff
@@ -60,61 +62,91 @@ import { SearchBar } from 'react-native-elements'
   showLoading
   platform="android"
   placeholder='Search' />
+
+<SearchBar
+  showLoading
+  platform="android"
+  cancelIcon={{ type: 'font-awesome', name: 'chevron-left' }}
+  placeholder='Search' />
 ```
 
 ### Props
 
-> This component inherits [all native TextInput props that come with a standard React Native TextInput element](https://facebook.github.io/react-native/docs/textinput.html), along with the following:
+> This component inherits all React Native Elements props, which means [all native TextInput props that come with a standard React Native TextInput element](https://facebook.github.io/react-native/docs/textinput.html), along with the following:
```

Comment:
Unclear what you mean by all React Native Elements props. Did you mean to say all React Native `<TextInput />` props?

Reply:
I forgot the `Input` word lol, should be "all React Native Elements `Input` props…"